File: domain/src/main/java/com/ortin/gesturetranslator/domain/repository/LoadImageRepository.java
package com.ortin.gesturetranslator.domain.repository;

import androidx.lifecycle.LifecycleOwner;

import com.ortin.gesturetranslator.domain.listeners.LoadImagesListener;

public interface LoadImageRepository {
    void loadImages(LoadImagesListener loadImagesListener, LifecycleOwner lifecycleOwner);

    void setStatusFlashlight(boolean mode);
}

File: domain/src/main/kotlin/com/ortin/gesturetranslator/domain/repository/LoadImageRepository.kt
package com.ortin.gesturetranslator.domain.repository

import androidx.lifecycle.LifecycleOwner
import com.ortin.gesturetranslator.domain.listeners.LoadImagesListener
import com.ortin.gesturetranslator.domain.models.CameraFacingSettings

interface LoadImageRepository {
    fun loadImages(
        loadImagesListener: LoadImagesListener,
        lifecycleOwner: LifecycleOwner,
        cameraFacing: CameraFacingSettings
    )

    fun setStatusFlashlight(mode: Boolean)
}
